pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -177,7 +177,7 @@ public SimpleLoadManagerImpl(PulsarService pulsar) {
         placementStrategy = new WRRPlacementStrategy();
         lastLoadReport = new LoadReport(pulsar.getWebServiceAddress(), pulsar.getWebServiceAddressTls(),
                 pulsar.getBrokerServiceUrl(), pulsar.getBrokerServiceUrlTls());
-        brokerHostUsage = new BrokerHostUsage(pulsar);
+        brokerHostUsage = new LinuxBrokerHostUsageImpl(pulsar);
```

Comment:
We could use the mbean based CPU usage for the "generic os"  implementation and keep using /proc/cpustat on Linux. 

For mac, we could omit he NIC traffic until we find a reasonable way to retrieve that.